.gitattributes

```diff
@@ -4,6 +4,7 @@ langref.html.in text eol=lf
 c_headers/ linguist-vendored

```

Comment:
Can probably fix these while you are here. Seems to require a `*` after the path.

See https://github.com/ziglang/zig/commit/83b79522d2c555d7bce8640f31b9ba20658cb021#commitcomment-32703676.